[docs] deprecate Classic Updates (#26962)
expo/expo: docs/pages/eas-update/faq.mdx

### Are Classic Updates still supported

> The Classic Updates service, which has been available before December 2021, is now deprecated. Users can no longer run `expo publish`, but existing apps will continue to receive Classic Updates that have already been published and are actively used.

Reviewer: ```suggestion
The Classic Updates service is now deprecated. It was available before December 202. You cannot run `expo publish` in newer projects. However, existing apps will continue to receive Classic Updates that have already been published and are actively used.
```

Let's add a distinction about the command no longer working in newer projects.

Author: We're turning off `expo publish` on Feb 12 for all projects, so it will apply to both old and new

Reviewer: In that case, we can discard my comment. 

I'd still recommend swapping  "Users" with "You" since users are mostly developers here and its kind of implicit (updated the suggestion above).

> The Classic Updates service, which has been available before December 2021, is now deprecated. Users can no longer run `expo publish`, but existing apps will continue to receive Classic Updates that have already been published and are actively used.

Author: ```suggestion
The Classic Updates service is now deprecated. It was available before December 2021, and `expo publish` cannot be run anymore. However, existing apps will continue to receive Classic Updates that have already been published and are actively used.
```

> We recommend users transition to EAS Update or use a [self-hosted update service](https://docs.expo.dev/eas-update/custom-updates-server).

Reviewer: ```suggestion
We recommend transitioning to EAS Update or using a [self-hosted update service](/eas-update/custom-updates-server).
```

- "users" are implicit so let's avoid using that
- Also we can use relative links for internal docs page
